ui/shared/components/form/Inputs.jsx

```diff
@@ -188,7 +193,7 @@ export class AddableSelect extends React.PureComponent {
 
   componentDidUpdate(prevProps) {
     const { options } = this.props
-    if (options.length !== prevProps.options.length) {
+    if (!_.isEqual(options, prevProps.options)) {
```

Comment:
AddableSelect was not updating when I changed the disabled/enabled property of an option, so I needed a deeper comparator than just checking the length of the array.

Reply:
More info: The MOI dropdown's options can "change" by being enabled/disabled, when different Gene Lists are selected.

This sort of change is not detected by comparing options.length, and requires a deeper comparator.
The current version of this branch uses our own seqr util function [propsAreEqual()](https://github.com/broadinstitute/seqr/pull/2790/files#diff-3de446691a6095256107e4e220e76df5e094fa5b63a9cd43b4311523486870abR18) instead of the lodash library.